{
  "action": "created",
  "data": {
    "issue": {
      "platform": "python",
      "lastSeen": "2020-04-28T13:56:05.903736Z",
      "numComments": 0,
      "userCount": 0,
      "culprit": "__main__ in <module>",
      "title": "Exception: Custom exception!",
      "id": "1635244907",
      "assignedTo": null,
      "logger": null,
      "type": "error",
      "annotations": [],
      "metadata": {
        "function": "<module>",
        "type": "Exception",
        "value": "Custom exception!",
        "filename": "trigger-exception.py"
      },
      "status": "unresolved",
      "subscriptionDetails": null,
      "isPublic": false,
      "hasSeen": false,
      "shortId": "ZULIP-INTEGRATION-TESTING-2",
      "shareId": null,
      "firstSeen": "2020-04-28T13:56:05.903736Z",
      "count": "1",
      "permalink": null,
      "level": "error",
      "isSubscribed": false,
      "isBookmarked": false,
      "project": {
        "platform": "python",
        "slug": "zulip-integration-testing",
        "id": "5216640",
        "name": "Zulip Integration Testing"
      },
      "statusDetails": {}
    }
  },
  "installation": {
    "uuid": "65027794-169a-4042-80d0-808ece377141"
  },
  "actor": {
    "type": "application",
    "id": "sentry",
    "name": "Sentry"
  }
}
